ccd-definition/CaseEvent.json

```diff
@@ -286,16 +286,18 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "otherProposalDecision",
+    "ID": "otherAllocationDecision",
     "Name": "Allocation decision",
     "Description": "Entering other proceedings and allocation proposals",
-    "DisplayOrder": 19,
+    "DisplayOrder": 1,
```

Comment:
As discussed please check with design which one should be first. Please move at the bottom of the file so it is together with the add hearing details, would be better to have got them grouped a bit by role (we are going to split this into multiple files per role anyway so will be good starting point)